app/utils/kcidb.py

```diff
@@ -122,6 +122,19 @@ def push_build(build_id, first, bq_options, db_options={}, db=None):
         })
         bq_data['revisions'] = [bq_revision]
 
+    files = {
+        'kernel_image': build[models.KERNEL_IMAGE_KEY],
+        'modules.tar.xz': build[models.MODULES_KEY],
```

Comment:
I think the key should be `modules` here, and the value will be `modules.tar.xz` typically but maybe something else or potentially nothing.

Reply:
Good point.  Updated changing the key to `modules`